cirq-core/cirq/__init__.py

```diff
@@ -65,8 +65,10 @@
 from cirq.circuits import (
     AbstractCircuit as AbstractCircuit,
     Alignment as Alignment,
+    BoxDrawCharacterSet as BoxDrawCharacterSet,
     Circuit as Circuit,
     CircuitOperation as CircuitOperation,
+    CIRCUIT_TYPE as CIRCUIT_TYPE,
```

Comment:
@daxfohl - it seems the automated tool added all names ever used for type annotations to the top `cirq` namespace.

Can you please review all modified `__init__.py` files and revert them to their initial shape?

The cirq-core/cirq/protocols/json_test_data/spec.py file appears to be also impacted due to additions to the `cirq` namespace and should be likely reverted as well.

Reply:
@pavoljuhas Is there any way we can keep these in this PR? They're all the classes returned by public functions. Reverting just these would probably be more work than the whole PR up to this point, and we likely should add them at some point.

Reply:
I might have oversold the amount of work that the AI did. It gave a decent starting point for an AST iterator and manipulator (which was pretty cool; I didn't know that library existed), but the actual manipulations were mostly hand written.

But anyway I agree with everything you say above. Per that interpretation of the issue, there were only a handful of cases remaining. They can be identified in this PR by the imports that were deleted. It's only a couple. So that issue can probably be closed too, or retitled to "migrate to future.annotations."

I'll go ahead and close this PR out. 

One open question is whether having everything consistently be annotated as 'cirq.*' would make docs more consistent. IDK how the doc generator works.